### ISSUE TITLE
cannot sign with newly added time-delayed proxy

### ISSUE THREAD

**Issue by Leemo94** [polkadot-js/apps#9582]
Hello, I was trying to add a delayed Any Proxy to a Pure Proxy and encountered a couple of bugs:
1. Cannot submit extrinsics from the Any Proxy, even though it shows up in the Proxy list in the UI
2. Cannot remove Any Proxy

I also tried to add a delayed Any Proxy to a standard account and I had similar results.

I recorded my experience here:
https://www.youtube.com/watch?v=CRukv-VmKTM

I explain my motivation for the acount setup at the end of the video.
I understand that the social recovery pallet is a thing (on Kusama), but nonetheless I think the behaviour in the video is a bug.

Thanks!

**Comment by Leemo94** [comment]
Oh, that's a shame that there's no support for delayed Proxies even though the parameter exists.

The main issue (outside of that) would be if you assign a delayed Proxy to an account, you then cannot remove it.

error: proxy.NotFound

**Comment by hitchhooker** [comment]
Cannot sign with a locked key pair, when sending tx with added pureproxy(600 timedelay).

**Comment by jacogr** [comment]
Nope. No support for time delayed proxies - it is not something I use, so never got to the top of the pile.
